pkg/kubelet/metrics/collectors/resource_metrics.go

```diff
@@ -23,6 +23,7 @@ import (
 	"k8s.io/component-base/metrics"
 	"k8s.io/klog/v2"
 	summary "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
+
```

Comment:
Can you remove this change? I don't think it's related to the PR.

Reply:
done